### ISSUE TITLE
A start() method to trigger a start using ref for custom start button component

### ISSUE THREAD

**Issue by tarun-soni** [TheWidlarzGroup/react-native-video-player#145]
I'm using a custom start button component and want to trigger the start using the ref, but I can see there are seek, stop pause, resume methods but not a start one. So is there any way to mimic the start() method?

Thanks in advance!

**Comment by alantoa** [comment]
Good idea, you can follow this library ` [react-native-video-player](https://github.com/alantoa/react-native-video-player)`, I will add later.

**Comment by tarun-soni** [comment]
@alantoa Sure!
